### PR TITLE
Issue #3104877 by angela_G, chr.fritsch, volkerk: Replace jquery.ui.sortable with SortableJs

### PR DESCRIPTION
**Pull request by chrfritsch**

### CODE REVIEW

File: js/select2.js [hunk 1]
```diff
@@ -46,12 +46,10 @@
...
           var $list = $(this).next('.select2-container').find('ul.select2-selection__rendered');
-          $list.sortable({
-            placeholder: 'ui-state-highlight select2-selection__choice',
-            forcePlaceholderSize: true,
-            items: 'li:not(.select2-search__field)',
-            tolerance: 'pointer',
-            stop: function () {
+          Sortable.create($list[0], {
+            draggable: 'li:not(.select2-search)',
+            forceFallback: true,
```
Comment (ol0lll) [inline js/select2.js:51]: I created a small demo at https://jsbin.com/rexadatola/edit?html,js,output
This seems to be a general problem with the combination of select2 and sortablejs.
We might want to add issues to these projects.
I am fine with using forceFallback for now, though

File: js/select2.js [hunk 2]
```diff
@@ -46,12 +46,10 @@
         if (Object.prototype.hasOwnProperty.call(config, 'ajax')) {
```
Comment (ol0lll) [inline js/select2.js]: This check is not sufficient since it applies sortable on single value select2 fields, like uid field (author).

**Review by ol0lll — CHANGES_REQUESTED** [review]
Tests need fixing ;-)

File: js/select2.js (hunk 2 above)
Comment (chrfritsch) [reply js/select2.js]: Added a check to ensure we use a multiple field.

**Comment by ol0lll** [comment]
Filed an issue there: https://github.com/SortableJS/Sortable/issues/1729

**Review by ol0lll — APPROVED** [review]
approve

**Pull request merged**